main.sh

```diff
@@ -238,11 +265,12 @@ for test in "$@"; do
     prepare_test_log_dir $test
     case $test in
         IOR)
-            start_server
-            start_agent
-	    create_pool
-            run_IOR
-            killall_proc $test
+            #start_server
```

Comment:
Why are these being disabled?

Reply:
I have not updated the dmg cmd to work with daos master. So the create pool is expected to fail.
I want to give Aruna the opportunity to update and enable IOR and MDTEST in her PR.